erpnext_com/www/docs/user/manual/en/accounts/accounting-dimensions.md

```diff
@@ -0,0 +1,31 @@
+<!-- add-breadcrumbs -->
+# Accounting Dimensions
+
+Accounting Dimensions are available in ERPNext from version 12 onwards.
+
+### 1.What is dimensional accounting?
+Dimensional accounting is basically tagging each transaction with appropriate dimensions like Branch, Business Unit, etc. This allows you to maintain each segment separately, thereby limiting the overall maintenance on GL accounts and your chart of accounts remains pure.
+
+In ERPNext you can create custom configurable accounting dimensions and use them in transactions and reports.
+
+### 2.How to create accounting dimensions in ERPNext?
+
+1. Go to **Accounting > Settings > Accounting Dimensions**.
+1. Select the reference document which you want to use as custom dimension.
```

Comment:
add: For example, if you select Cost Center, the dimension will be based on Cost Center.